src/adv.py

```diff
@@ -49,3 +57,32 @@
 # Print an error message if the movement isn't allowed.
 #
 # If the user enters "q", quit the game.
+
+directions = ['n', 'e', 's', 'w']
+
```

Comment:
If you're hard-coding sequences like this, and don't intend to alter them in any way, you can use a tuple instead of a list, which is slightly quicker under the hood.